web/bfex/components/scraper/scraper_type.py

```diff
@@ -1,6 +1,9 @@
 from enum import Enum
 
 class ScraperType(Enum):
+        """ Enum class to keep track of all the different types of web pages 
```

Comment:
This could be one line.

Reply:
then it goes over pep8 line max, no thank you